data/analysis/disability_weights.csv

```diff
@@ -1,10 +1,35 @@
 disease,weight
 alzheimer,0.298333333
```

Comment:
The new diseases, including `ischemicheartdisease` are not present here. Should probably be added.